packages/recommendations-core/src/getFrequentlyBoughtTogether.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  getRecommendations,
+  GetRecommendationsProps,
+} from './getRecommendations';
+
+export type GetFrequentlyBoughtTogetherProps<TObject> = Omit<
+  GetRecommendationsProps<TObject>,
+  'model' | 'fallbackFilters'
+>;
+
+export function getFrequentlyBoughtTogether<TObject>(
```

Comment:
this name feels a little bit like an unfinished sentence, since it's already implying e-commerce with the word "bought", what about "getFrequentlyBoughtTogetherProducts"?

Reply:
I agree—it would also be more consistent with "getRelated**Products**". The thing is it's been branded like that every where else on the AI team side. I'll catch up with them to see what they think about that.